java/maintenance-scheduling/src/main/java/org/acme/maintenancescheduling/solver/MaintenanceScheduleConstraintProvider.java

```diff
@@ -22,8 +22,8 @@ public Constraint[] defineConstraints(ConstraintFactory constraintFactory) {
         return new Constraint[] {
                 // Hard constraints
                 crewConflict(constraintFactory),
-                readyDate(constraintFactory),
-                dueDate(constraintFactory),
+                minDate(constraintFactory),
+                maxDate(constraintFactory),
```

Comment:
Shouldn't this be `minStartDate` and `maxEndDate`?

Reply:
The maintenance module uses a `LocalDate`, while the food packing and VRP modules use a `LocalDateTime`. Hence, we will use `minStartDate` for the first one and `minStartDateTime` for the others.